themes/10up-theme/postcss.config.js

```diff
@@ -1,8 +1,27 @@
-module.exports = {
-	plugins: {
-		'postcss-import': {},
-		'tailwindcss/nesting': {},
-		tailwindcss: {},
-		autoprefixer: {},
-	},
+/* eslint-disable global-require */
+
+module.exports = () => {
+	const config = {
+		plugins: [],
+	};
+
+	// Only load postcss-editor-styles plugin when we're processing the editor-style.css file.
+
+	config.plugins.push(require('postcss-import'));
```

Comment:
@Nikki-Jones Why are we using push here? I'm not sure what value the extra function call gives us here when 3/4 are one line. Plus it's a pretty well established convention to just define in the array literal, especially if we're not conditionally adding anything to the array.